src/coreclr/src/vm/readytoruninfo.h

```diff
@@ -48,13 +75,18 @@ class ReadyToRunInfo
 
     PTR_PersistentInlineTrackingMapR2R m_pPersistentInlineTrackingMap;
 
-    ReadyToRunInfo(Module * pModule, PEImageLayout * pLayout, READYTORUN_HEADER * pHeader, AllocMemTracker *pamTracker);
-
 public:
+    ReadyToRunInfo(Module * pModule, PEImageLayout * pLayout, READYTORUN_HEADER * pHeader, NativeImage * pNativeImage, AllocMemTracker *pamTracker);
+
     static BOOL IsReadyToRunEnabled();
 
     static PTR_ReadyToRunInfo Initialize(Module * pModule, AllocMemTracker *pamTracker);
 
+    bool IsComponentAssembly() const { return m_isComponentAssembly; }
+    const ReadyToRunCoreInfo * GetComponentInfo() const { return &m_component; }
+    const ReadyToRunCoreInfo * GetCompositeInfo() const { return m_pComposite; }
```

Comment:
This function isn't used.

Reply:
Refactored per your other suggestion in 11th commit.